src/Data/IPPInventoryAdjustment.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace QuickBooksOnline\API\Data;
+
+/**
+ * @xmlNamespace http://schema.intuit.com/finance/v3
+ * @xmlType Transaction
+ * @xmlName IPPInventoryAdjustment
+ * @var IPPInventoryAdjustment
+ * @xmlDefinition 
+				Product: QBO
+				Description: The Inventory Adjustment request element
+			
+ */
+class IPPInventoryAdjustment
+	extends IPPTransaction	{
+
+		/**                                                                       
+		* Initializes this object, optionally with pre-defined property values    
+		*                                                                         
+		* Initializes this object and it's property members, using the dictionary
```

Comment:
"It's" is incorrect here. It should be "its".
```suggestion
		* Initializes this object and its property members, using the dictionary
```